_overviews/contribute/contribution-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Contribute to Scala 3
+description: This page describes the format of the contribution guide for the Scala 3 compiler.
+num: 14
```

Comment:
For internal redirections (ie, not redirections to https://dotty.epfl.ch but to https://docs.scala-lang.org), it is better to use `redirect_from`:

```suggestion
num: 14
redirect_from: /scala3/guides/contribution/contribution-intro.html
```

(and remove the `redirect_to` in the file `contribution-intro.md`)

Reply:
Ah, good catch, that link needs to be updated as well to point to `_overviews/contribute/scala3.md` (following the other comment).